commands/system/help.js

```diff
@@ -9,75 +10,146 @@ class Help extends Command {
+    if (this.client.commands.has(type) || this.client.aliases.has(type)) {
+      const cm = this.client.commands.get(type) || this.client.commands.get(this.client.aliases.get((type)));
       if (level < this.client.levelCache[cm.conf.permLevel]) return;
-      embed.setTitle(cm.help.name.toProperCase())
+      helpembed.setTitle(cm.help.name)
         .addField("Command description", cm.help.description)
         .addField("Command usage", `\`${cm.help.usage}\``)
-        .addField("Command aliases", cm.conf.aliases.length == 0 ? "None" : cm.conf.aliases.join(", ") );
+        .addField("Command aliases", cm.conf.aliases.length ? "None" : cm.conf.aliases.join(", "));
+    }
+    const msg2 = await message.channel.send(helpembed);
+    const totalpages = Math.ceil(num / perpage);
+    if (!message.guild.me.hasPermission(["MANAGE_MESSAGES"])) {
+      await message.channel.send("I don't have permission to remove reactions, please do this manually.");
+    }
+
+    if (msg2.embeds[0].title && msg2.embeds[0].title.includes("Page") && totalpages > 1) {
+      for (const emoji of EMOJIS) await msg2.react(emoji);
     }
-    
-    if (!embed.fields[0]) return;
 
-    return message.channel.send(embed);
+    const select = msg2.createReactionCollector(
+      (reaction, user) => EMOJIS.includes(reaction.emoji.name) && user.id === message.author.id,
+      { time: 30000 }
+    );
+
+    let on = false;
+    select.on("collect", async (r) => {
+      const currentpage = Number(msg2.embeds[0].title.split(" ")[1].split("/")[0]);
+      switch (r.emoji.name) {
+        case "▶":
+          this.pages(message, msg2, currentpage + 1, sorted, type, level, r, "forward");
+          break;
+        case "◀":
+          this.pages(message, msg2, currentpage - 1, sorted, type, level, r, "backward");
+          break;
+        case "⏮":
+          this.pages(message, msg2, 1, sorted, type, level, r);
+          break;
+        case "⏭":
+          this.pages(message, msg2, totalpages, sorted, type, level, r);
+          break;
+        case "⏹":
+          select.stop();
+          r.message.reactions.removeAll();
+          break;
+        case "🔢": {
+          if (on) return;
+          on = true;
+          await r.message.channel.send(`Please enter a selection from 1 to ${totalpages}`);
+          const whichpage = await message.channel.awaitMessages(m => !isNaN(m.content) && m.author.id === message.author.id, {
+            max: 1,
+            time: 300000,
+            errors: ["time"]
+          });
+          const pagenumber = Number(whichpage.first().content);
+          this.pages(message, msg2, pagenumber, sorted, type, level, r);
+          on = false;
+          break;
+        }
+      }
+    });
+
+    select.on("end", (r, reason) => {
+      if (reason === "time") {
+        msg2.reactions.removeAll();
+      }
+    });
   }
 }
 
-module.exports = Help;
\ No newline at end of file
+Array.prototype.unique = function() {
```

Comment:
**Big nope** from me to this, this is bad practise at all but if you actuall need to do that add this inside the function.js file atleast.